shared/common-adapters/input.native.js

```diff
@@ -267,12 +241,6 @@ class Input extends Component<Props, State> {
       ...commonProps,
       multiline: false,
       style: {...singlelineStyle, ...this.props.inputStyle},
```

Comment:
Don't we use password entries and password visible entries?

Reply:
Are you referring to the `type` field? I couldn't find a `type` field for TextInput (looked on https://facebook.github.io/react-native/docs/textinput.html and https://facebook.github.io/react-native/docs/view.html#props ) -- afaik, `secureTextEntry` is what controls this.